docs/AOT.md

```diff
@@ -17,10 +17,11 @@ bundles are object files that can be linked with some executable. On other
 architectures, the bundle may look completely different.
 
 This document demonstrates how to produce a bundle for the host CPU using the
-'loader' tool.  We use the flag `-emit-bundle` to specify the output directory.
+'ImageLoader' tool.  We use the flag `-emit-bundle` to specify the output
```

Comment:
nit: there are also some entries of loader in our AOT examples, can you update those to be consistent?
examples//bundles/*

Reply:
Whoops, good catch!